src/assets/sass/_header.scss

```diff
@@ -0,0 +1,109 @@
+@media only screen and (min-width: 769px){
+    .header-container {
+        display: block;
+    }
+}
+.header-container .header-main {
+    color: #0b0c0c;
+    padding: 0.5rem 0px;
+    border-bottom: 6px solid #008938;
+    background: #FFF !important;
+}
+
+.header-div-container {
+    width: 100%;
```

Comment:
I believe that we can remove this, isn't it?

Reply:
As per our discussion, there is no harm in having this. We can ignore this.

Reply:
we must keep it as we are using box model. the width covers upto the border area.